website/source/docs/agent/options.html.md

```diff
@@ -562,17 +562,12 @@ default will automatically work with some tooling.
      all of the configured ACL tokens for the agents usage.
 
         * <a name="acl_tokens_master"></a><a href="#acl_tokens_master">`master`</a> - Only used
-          for servers in the [`primary_datacenter`](#primary_datacenter). This token will be created with management-level
-          permissions if it does not exist. It allows operators to bootstrap the ACL system
-          with a token Secret ID that is well-known.
+          for servers in the [`primary_datacenter`](#primary_datacenter). Initially, this token will be created with management-level
+          permissions. This process allows operators to bootstrap the ACL system with a token Secret ID that is well-known.
           <br/><br/>
-          The `master` token is only installed when a server acquires cluster leadership. If
-          you would like to install or change the `acl_master_token`, set the new value for `master`
-          in the configuration for all servers. Once this is done, restart the current leader to force a
-          leader election. If the `master` token is not supplied, then the servers do not create a master
-          token. When you provide a value, it should be a UUID. To maintain backwards compatibility
-          and an upgrade path this restriction is not currently enforced but will be in a future major
-          Consul release.
+          The token can be created with the [HTTP API](/api/acl/acl.html), [Consul CLI](/docs/commands/acl.html), or by providing a unique UUID as the value. When the operator provides a unique UUID, Consul will automatically assign it the "global-management" policy. Additionally, when providing a value, it should be a UUID to maintain backwards compatibility and an upgrade path this restriction is not currently enforced but will be in a future major
```

Comment:
I'm not sure how you could create this with the CLI? Do you mean by using the bootstrap workflow? If yes then I think that's confusing since the master token in config is an alternative mechanism to the online API/CLI bootstrap workflow?

We could maybe say here "If you would like to avoid having a root secret in your server config files you may alternatively use the [API-based bootstrap](link to ACL bootstrap guide)." or something?

Hmm actually I think that's really the only bit that changed - I was going to suggest an edit but it wuld be pretty close to reverting to the old wording - can you maybe explain what you thought was confusing about the previous wording?